lib/test.rb

```diff
@@ -21,7 +21,7 @@ class Test
 
     attr_reader :log_root, :category, :name, :steps
 
-    def initialize(argument, tap: nil, skip_setup: false, skip_cleanup_before: false, skip_cleanup_after: false)
+    def initialize(argument, tap: nil, skip_setup: false, skip_cleanup_before: false, skip_cleanup_after: false, git: "git")
```

Comment:
```suggestion
    def initialize(argument, git:, tap: nil, skip_setup: false, skip_cleanup_before: false, skip_cleanup_after: false)
```

Can also remove any other arguments that are always set.